_posts/2022-11-22-trino-summit-2022-state-of-trino-keynote-recap.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "Trino Summit 2022 recap: The state of Trino"
+author: "Martin Traverso, Dain Sundstrom, David Phillips, Cole Bowden"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2022/keynote-header.jpeg
+---
+
+To kick off the [Trino Summit 2022]({% post_url 2022-11-21-trino-summit-2022-recap %}),
+we heard from Trino co-creators Martin Traverso, Dain Sundstrom, and David
+Phillips. Martin gave a talk on the state of Trino and project plans for 2023,
+then opened the floor to questions from the community. Check out the recording
+of the talk below, or read on if you're only interested in the highlights.
+
+<!--more-->
+
+## The State of Trino talk
+
+{% youtube mUq_h3oArp4 %}
+
+[Download a PDF of the slides here.](https://www.starburst.io/wp-content/uploads/2022/11/State-of-Trino-Nov-2022.pdf)
+
+## Recapping
+
+So what *has* happened in Trino over the last year?
+
+* We celebrated Trino's 10th birthday!
+* It was the busiest year in project history, with 600+ contributors, 4000+
+  commits, and near-weekly releases.
+* Tons of new features were added, including `MERGE`, JSON functions, table
+  functions, fault-tolerant execution (look forward to a lot of talking about it
```

Comment:
get rid of this promise for a future post

Reply:
I like it! And we're going to talk about it, I think it's ok to make a promise if we're 100% going to follow-through on it.

Reply:
we are not .. just look at the blog post we had in the pipeline for a few months now